src/hrc721.spec.ts

```diff
@@ -39,4 +39,18 @@ describe('HarmonyProvider', () => {
       expect(provider.balanceOf('')).to.be.rejectedWith(Error)
     })
   })
+
+  describe('ownerOf', () => {
+    it('should returns the owner of the tokenId token.', async () => {
+      const owner = await provider.ownerOf('1')
+      expect(owner).to.exist
+      expect(owner).to.not.be.null
+      expect(owner).to.not.be.undefined
+      expect(owner).to.be.equals(RESULT_TEST_ADDRESS)
+    }).timeout(5000)
```

Comment:
maybe we can add a test of what happens when we provide a token with no owner and a non existent token

Reply:
Roger that!